### DISCUSSION TITLE
God job

### DISCUSSION THREAD

**Discussion by gong88888888** [GaParmar/img2img-turbo#19] [General]
Thanks to the excellent work of the authors, I have completed the initial deployment and preliminary testing of this project on windows10. I hope the author will continue his efforts.

**Comment by shuizk** [comment]
After trying clear to rainy, I have to say that the result is not satisfying. All I can see is that the hue has changed. Details such as raindrops were not added, so it not looks real.

**Reply by GaParmar** [reply]
This is slightly surprising to me. What dataset did you try this on?
